### PR TITLE
Gh pages integration card9

### PR DESCRIPTION
**Pull request by ianleeduckworth**

### CODE REVIEW

File: docs/documentation/configuration.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    <li><a href="./EzPerformanceMonitor.html">EzPerformanceMonitor</a></li>
+    <li><a href="./EzProcess.html">EzProcess</a></li>
+    <li class="leftnav-active"><a href="./configuration.html">Configuration</a></li>
+</ul>
+
+<div class="main-documentation">
+    <h1>EazyE2E Configuration</h1>
+    <p>
+        EazyE2E provides the ability to configure various ways that the framework will function.  Any of the values 
+        below can be placed in the applications app.config file in the &lt;appSettings&gt; section.  Note that these
```
Comment (hopeduckworth) [inline docs/documentation/configuration.html]: Revise "applications" to "application's"

File: docs/documentation/configuration.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        <tr>
+            <td>int</td>
+            <td>&lt;add key="MaximumMemoryProfileTime" value="600000"/&gt;</td>
+            <td>int.MaxValue</td>
+        </tr>
+    </table>
+
+    <h3>TimeBetweenMouseEvents</h3>
+    <p>
+        The time inbetween mouse events (in milliseconds); used by EzMouseFunctions.  This value is used to determine how much time inbetween various
```
Comment (hopeduckworth) [inline docs/documentation/configuration.html]: "inbetween" should be "in between"- same comment immediately below

**Review by hopeduckworth — COMMENTED** [review]
Just a few minor typos

File: docs/documentation/configuration.html (hunk 1 above)
Comment (ianleeduckworth) [reply docs/documentation/configuration.html]: Changed text to "application's"

File: docs/documentation/configuration.html (hunk 2 above)
Comment (ianleeduckworth) [reply docs/documentation/configuration.html]: Changed both instances to "in between"

**Review by hopeduckworth — APPROVED** [review]

**Pull request merged**